.gitlab-ci.yml

```diff
@@ -228,9 +228,9 @@ test-dependency-rules:
 unleash-check:
   stage:                           test
   <<:                              *docker-env
-  only:
-    - master
-    - tags
+  # only:
```

Comment:
How is this related to PR?

Reply:
it isn't. It is here to trigger the specific CI job to fix it. Once the CI passed, this is reverted.